bot/seasons/halloween/__init__.py

```diff
@@ -3,14 +3,8 @@
 
 class Halloween(SeasonBase):
     name = "halloween"
+    greeting = "Happy Halloween!"
     start_date = "01/10"
     end_date = "31/10"
     bot_name = "Spookybot"
```

Comment:
Position of `bot_name` is inconsistent between seasons, with the Christmas cog it is with the avatar but here it is at the top of the class.

Reply:
If we're going to end up changing the ordering of the season dataclasses, we're better off properly establishing proper logical grouping then.

May I suggest:
```py

    # strings
    name = "halloween"
    bot_name = "Spookybot"
    greeting = "Happy Halloween!"

    # dates
    start_date = "01/10"
    end_date = "31/10"

    # resources
    icon = "/logos/logo_seasonal/halloween/spooky.png"
```